package.json

```diff
@@ -150,7 +150,7 @@
     "@typescript-eslint/parser": "5.30.7",
     "babel-eslint": "10.1.0",
     "babel-plugin-transform-imports": "2.0.0",
-    "cross-env": "7.0.3",
+    "cross-env": "^7.0.3",
```

Comment:
why have you committed this changes to `package.json`?

Reply:
I will undo it.